tests/tools/conftest.py

```diff
@@ -0,0 +1,34 @@
+from typing import TYPE_CHECKING
+
+import pytest
+
+if TYPE_CHECKING:
+    from types import ModuleType
+    from typing import Any, Dict
+
+
+@pytest.fixture
+def tool(request: "Dict[str, Any]") -> "ModuleType":
+    """Import a tool as a module."""
+    from copy import deepcopy
+    import importlib
+    from pathlib import Path
+    import sys
+
+    original_sys_path = deepcopy(sys.path)
+    original_tool_path: Path = (
+        Path(__file__).resolve().parent.parent.parent / "tools" / request.param
```

Comment:
Why do we have to add the path specifically? Aren't the tools in sys.path when installed through emmontopy?

Reply:
The tools edit `sys.path`, therefore, I'm storing and restoring `sys.path` to it's original value before and after a test (that uses this fixture).
We should reconsider the way this is done, to be honest, either relying on non-Python scripts to run the tools if it's not desirable to install the package and otherwise simply use `setuptools` to define the tools. This will auto-generate `sh` files in a place that will be added to the environment's `PATH` variable automatically. Adding it manually is --- user unfriendly :)

Reply:
Thank you for the explanation.